packages/emitter-framework/src/python/components/enum-declaration/enum-declaration.test.tsx

```diff
@@ -0,0 +1,246 @@
+import { Tester } from "#test/test-host.js";
+import { StatementList } from "@alloy-js/core";
+import { d } from "@alloy-js/core/testing";
+import { t, type TesterInstance } from "@typespec/compiler/testing";
+import { beforeEach, describe, expect, it } from "vitest";
+import { efRefkey } from "../../../../src/python/utils/refkey.js";
+import { getOutput } from "../../test-utils.js";
+import { EnumDeclaration } from "./enum-declaration.js";
+
+let runner: TesterInstance;
+
+beforeEach(async () => {
+  runner = await Tester.createInstance();
+});
+
+describe("Python Enum Declaration", () => {
+  it("takes an enum type parameter", async () => {
+    const result = await runner.compile(t.code`
+      enum ${t.enum("Foo")} {
+        one: 1,
+        two: 2,
+        three: 3
+      }
+    `);
+    const output = getOutput(runner.program, [<EnumDeclaration type={result.Foo} />]);
+
+    expect(output).toRenderTo(d`
+      from enum import IntEnum
+
+      class Foo(IntEnum):
+        ONE = 1
+        TWO = 2
+        THREE = 3
+
+
+    `);
+  });
+
+  it("adds JSDoc from TypeSpec", async () => {
+    const result = await runner.compile(t.code`
+      /**
+       * This is a test enum
+       */
+      enum ${t.enum("Foo")} {
+        @doc("This is one")
+        one: 1,
+        two: 2,
+        three: 3
+      }
+    `);
+    const output = getOutput(runner.program, [<EnumDeclaration type={result.Foo} />]);
+
+    expect(output).toRenderTo(d`
+      from enum import IntEnum
+
+      # This is a test enum
```

Comment:
This doc should be inside the class body, no? Don't we have a special Alloy component for that?